style.css

```diff
@@ -440,3 +453,476 @@ li {
 .form-body {
   background-color: #fff;
 }
+
+/* Desktop */
+
+@media only screen and (min-width: 900px) {
+  .nav-mobile {
+    display: none;
+  }
```

Comment:
- [ ] The desktop breakpoint is currently too low. As per requirement, it should be a value between `992px` and `1100px`. You're currently using `900px`, please make sure to address this issue.